Tests/ComposableArchitectureTests/ViewStoreTests.swift

```diff
@@ -137,7 +137,7 @@ final class ViewStoreTests: XCTestCase {
     XCTAssertNoDifference(results, [0, 1])
   }
 
-  #if compiler(>=5.5)
+  #if canImport(_Concurrency)
```

Comment:
should this be kept too?

```suggestion
  #if compiler(>=5.5) && canImport(_Concurrency)
```